app/controllers/application_controller.rb

```diff
@@ -37,7 +37,9 @@ def after_sign_in_path_for(user)
     end
   end
 
-  private
+  def current_user
```

Comment:
Would we have issues moving this method under the `protected` session?
Can we create a spec to ensure the object returned will be always a decorated user?